java/ql/lib/ext/io.undertow.server.handlers.resource.model.yml

```diff
@@ -0,0 +1,6 @@
+extensions:
+  - addsTo:
+      pack: codeql/java-all
+      extensible: sinkModel
+    data:
+      - ["io.undertow.server.handlers.resource", "URLResource", True, "URLResource", "(URL,String)", "", "Argument[0]", "request-forgery", "ai-manual"]
```

Comment:
Judging by the methods in the [documentation](https://undertow.io/javadoc/2.0.x/io/undertow/server/handlers/resource/URLResource.html) and some [code search](https://github.com/search?type=code&q=lang%3Ajava+%22import+io.undertow.server.handlers.resource.URLResource%3B%22), this looks like a class you use to expose (i.e. serve) some resource, not to request it. So it shouldn't qualify as `request-forgery`.

Reply:
cd765e7 thank you, I've removed the model!